check_pgactivity

```diff
@@ -5551,6 +5558,18 @@ sub check_streaming_delta {
 
     @rs = @{ query_ver( $hosts[0], %queries ) };
 
+    my $index = 0;
```

Comment:
If you keep "#of slaves" with the excluded connections then what's the point to this change ?
I think if we take the decision to exclude something it should be counted out of total of connected slaves.